sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -151,6 +184,44 @@ def _predict(self, fh=None, X=None):
         y_pred.name = self._y.name
         return y_pred
 
+    def _predict_var(self, fh=None, X=None, cov=False):
+        """Compute the variance at each forecast horizon."""
+        if self.prediction_intervals is False:
```

Comment:
```suggestion
        if not self.prediction_intervals:
```

Reply:
@fnhirwa is it not something ruff should capture, or is some rule disabled? It definitely gets flagged by older linters.

Reply:
FYI @fnhirwa re `ruff`